modules/custom_operations/pyproject.toml

```diff
@@ -17,8 +18,7 @@ classifiers = [
 ]
 
 dependencies = [
-    "openvino",
-    "numpy"
+    "openvino==2023.3.0"
```

Comment:
maybe we need to add strict version only for release branches?
currently people can build extension on top of 2023.2 release